packages/documentation-site/patternfly-docs/content/design-guidelines/components/button/button.md

```diff
@@ -75,9 +75,7 @@ Links buttons are labeled buttons with no background or border. Link buttons can
 
 While all link buttons lack borders, they can still vary in appearance. Link buttons that navigate users to another page within the same window should be presented as stand-alone text. Link buttons that navigate users to an external page may include an icon on the left or right of the text to further emphasize the action. For example, you could add an external link icon to show that clicking on it will navigate users externally to another website or application.
 
-<img src="./img/link-left.png" alt="Example of link button with icon on the left" width="143"/>
-
-<img src="./img/link-right.png" alt="Example of link button with icon on the right" width="143"/>
+<img src="./img/external_links.png" alt="Example of link button with icon on the left and the right" width="218"/>
```

Comment:
```suggestion
<div class="ws-docs-content-img">
<img src="./img/external_links.png" alt="Example of link button with icon on the left and the right" width="218"/>
</div>
```

Reply:
If you add this import line at the top of the file: 

`import '../components.css';`

You should be able to use this class for all images, which centers them / controls max width! I would assume that you can keep the width you're manually setting on these individual/smaller images still?

Reply:
Sorry in advance that this would require a handful of updates -- I couldn't leave GH suggestions on all of them!